content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -3,6 +3,17 @@
 DVC provides commands to list, tabulate and contrast experiments. Let's see how
 they can help you streamline the experimentation process.
 
+<admon type="tip">
+
+**New!** You can compare your ML experiments with DVC directly [from Visual
+Studio Code], a leading IDE in the industry.
```

Comment:
minor: we are trying to avoid "marketing" language in docs. I would remove `a leading IDE in the industry`.

Reply:
No problem. Just borrowed this from https://dvc.org/doc/start/experiment-management/experiments, which we will hopefully update soon.